app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    comfort_class: int
+    clean_mark: int
+    brand: str
+
+    def __init__(self, c_class: int, c_mark: int, car_brand: str):
```

Comment:
use more descriptive names of parameters. The person who will use your class will not understand what he should pass to it